.docker/search/log4j2.properties

```diff
@@ -0,0 +1,51 @@
+status = error
```

Comment:
could you link at the top the source of this, if there is a Github repository or a path in the Docker image it was copied from? Helps with updating

Reply:
Not necessary if it's just the original `/usr/share/elasticsearch/config/log4j2.properties`

Reply:
Correct. This is a one for one copy of the original with the `appender.rolling.strategy` section added but I will add comments for clarification.